spec/computer_player_spec.rb

```diff
@@ -16,11 +16,22 @@ def computer_player_setup
       computer_player = computer_player_setup
       allow(computer_player).to receive(:sleep)
 
-      board = Board.new([1, 2, 3])
+      board = Board.new(['x', 'o', 3, 4, 5, 6, 7, 8, 9])
 
-      choose_move = computer_player.choose_move(board)
+      choose_move = computer_player.choose_move(board, 'x', 'o')
 
-      expect(choose_move).to be_between(1, 3)
+      expect(choose_move).to be_between(3, 9)
+    end
+    
+    it 'returns either 1, 3, 7, 9 when the board is empty' do
```

Comment:
Shouldn't it always return the same number when empty?

Reply:
@Kotauror, the unbeatable player always picked 1 if the board was empty. 
For variation, I included a method where the unbeatable plays a corner move (1, 3, 7 or 9) if the board is empty.
Do you think I should remove this?

Reply:
Oh, I see! Sorry, I missed it - in such case the test is correct. I think the feature is cool, makes the game more interesting! :) 

On the other hand, the feature is a bit surprising. I think that in the future, a good practice would be to tell the client about your idea (if the idea wasn't "ordered" by the client) before you implement it so that the client is not surprised. In "pure" minimax, the computer always returns the same position and getting different ones could be a bit confusing. That will make the client sure they pay for what they really want and need.

Reply:
Thank you :) I'll take this into consideration